doc/rose-rug-quiz.html

```diff
@@ -946,7 +943,7 @@ <h2 id="questions">Questions</h2>
 
             <p class="answers C"><em><samp>[cylc]</samp> <samp>[[event
             hooks]]</samp>...</em> - this makes sure your suite does not become
-            a long-lived zombie. The <samp>command scripting</samp> answer
+            a long-lived zombie. The <samp>script</samp> answer
```

Comment:
Not sure if the question no longer applies in the context of you having removed all the event hooks?